README.md

```diff
@@ -1,87 +1,105 @@
-# Transparent Release of Binaries
+# The Transparent Release Project
+
+Transparent Release (not really the best name for the project) builds on the idea of binary
+transparency and expands in a few directions, as described below. 
+
+## Release Transparency
+
+The goal of release transparency is to generate and transparently publish ideally-verifiable metadata about a released binary. We refer to this kind of metadata as the endorsement statement. It can be used in scenarios where the identity of a binary must be verified before use. One prominent use case is remote attestation. See Oak’s remote attestation protocol for instance.
+
+The life cycle of an endorsement statement has three phases as shown in the diagram below:
+
+![The journey of a software binary](docs/images/release-transparency.png)
+
+### The pre-release phase
+In this phase, a trusted builder, for instance [the generic slsa-generator GitHub workflow](https://github.com/slsa-framework/slsa-github-generator/blob/de4491844e9be4184f786666af40f5b1b8e7ddc0/internal/builders/generic/README.md),
+is used to generate the binary and a signed SLSA provenance statement about it. A provenance
```

Comment:
Suggestion to active:

In this phase a Trusted Builder, for instance ..., generates a Binary with a corresponding SLSA Provenance statement from Sources.